Sources/Jobs/Scheduler/Schedule+crontab.swift

```
@@ -0,0 +1,127 @@
+        switch (schedule.day, schedule.date) {
+        case (.selection, .specific), (.selection, .selection), (.specific, .selection):
+            throw ScheduleError(
+                "Schedule does not support a combination of date and weekday where one is a selection of values and the other is not the wildcard '*'"
+            )
+        default:
+            break
+        }
+        return schedule
+    }
+
+    static func parse<Value>(_ entry: Substring, range: ClosedRange<Int>, transform: (Int) throws -> Value) throws -> Parameter<Value> {
+        let numberRegex = Regex {
+            Anchor.startOfLine
+            Capture {
+                OneOrMore(.digit)
+            } transform: {
+                Int($0)!
+            }
+            Anchor.endOfLine
+        }
+        let rangeRegex = Regex {
+            Anchor.startOfLine
+            Capture {
+                OneOrMore(.digit)
+            } transform: {
+                Int($0)!
+            }
+            "-"
+            Capture {
+                OneOrMore(.digit)
+            } transform: {
+                Int($0)!
+            }
+            Anchor.endOfLine
+        }
+        let everyRegex = Regex {
+            Anchor.startOfLine
+            "*/"
+            Capture {
+                OneOrMore(.digit)
+            } transform: {
+                Int($0)!
+            }
+            Anchor.endOfLine
+        }
+        let selectionRegex: Regex = Regex {
+            Anchor.startOfLine
+            Capture {
+                OneOrMore {
+                    OneOrMore(.digit)
+                    ","
+                }
+                OneOrMore(.digit)
+            } transform: {
+                $0.split(separator: ",").map { Int($0)! }
+            }
+            Anchor.endOfLine
+        }
+        if entry == "*" { return .any }
+        if let values = try? numberRegex.wholeMatch(in: entry) {
```

is the exception not swallowed here?